worker-jobtracking/src/main/java/com/hpe/caf/worker/jobtracking/JobTrackingWorkerReporter.java

```diff
@@ -99,14 +100,42 @@ public JobTrackingWorkerReporter() throws JobReportingException {
     /**
      * Reports the percentage completed of the specified task id.
      *
-     * @param jobTaskId identifies the job task whose progress is to be reported
+     * @param jobTaskId identifies the job task in progress
      * @param estimatedPercentageCompleted an indication of progress on the job task
      * @throws JobReportingException if a failure occurs in connecting or reporting to a Job Database
      */
     @Override
     public void reportJobTaskProgress(final String jobTaskId, final int estimatedPercentageCompleted) throws JobReportingException
     {
-        LOG.trace("Received progress update message for task {}; taking no action", jobTaskId);
```

Comment:
If the `estimatedPercentageCompleted` is 0% then don't bother making the database call.

Reply:
done